src/cosmic/cosm.rs

```diff
@@ -755,7 +755,7 @@ impl Cosm {
                 let state = Orbit::cartesian(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, datetime, target_frame);
                 Ok(-self.try_frame_chg(&state, frame)?)
             }
-            LightTimeCalc::LightTime | LightTimeCalc::Abberation => {
+            LightTimeCalc::LightTime | LightTimeCalc::Aberration => {
```

Comment:
Same comment

Reply:
Resolved in commit 96a6ad32a5d0dc19579112d430d2411efecbbf96.